var/spack/repos/builtin/packages/amdscalapack/package.py

```diff
@@ -46,12 +46,32 @@ class Amdscalapack(ScalapackBase):
     version("2.2", sha256="2d64926864fc6d12157b86e3f88eb1a5205e7fc157bf67e7577d0f18b9a7484c")
 
     variant("ilp64", default=False, description="Build with ILP64 support")
+    variant(
+        "enable-aocl-blas-lapack",
+        default=True,
+        when="@4.1:",
+        description=(
+            "Enables tight coupling with AOCL BLAS/LAPACK library in order to use AOCL "
+            "internal routines"
+        ),
+    )
```

Comment:
Same as above: the variant is not needed, since we can take decisions internally based on the providers of blas and lapack.